Include/internal/pycore_symtable.h

```diff
@@ -15,11 +15,21 @@ typedef enum _block_type {
     // Used for annotations if 'from __future__ import annotations' is active.
     // Annotation blocks cannot bind names and are not evaluated.
     AnnotationBlock,
-    // Used for generics and type aliases. These work mostly like functions
-    // (see PEP 695 for details). The three different blocks function identically;
-    // they are different enum entries only so that error messages can be more
-    // precise.
-    TypeVarBoundBlock, TypeAliasBlock, TypeParamBlock
+
+    // The following blocks are used for generics and type aliases. These work
+    // mostly like functions (see PEP 695 for details). The three different
+    // blocks function identically; they are different enum entries only so
+    // that error messages can be more precise.
+
+    // The block to enter when processing a "type" (PEP 695) construction,
+    // e.g., "type MyGeneric[T] = list[T]".
+    TypeAliasBlock,
+    // The block to enter when processing a "generic" (PEP 695) object,
+    // e.g., "def foo[T](): pass" or "class A[T]: pass".
+    TypeParametersBlock,
+    // The block to enter when processing a single "type variable" in the
```

Comment:
This doesn't tell the whole story as there may be more than one block per type variable (`def f[T: A = B](): pass`).
```suggestion
    // The block to enter when processing the bounds, constraints, or
    // default value of a single "type variable" in the
```

Reply:
Thank you, I wasn't precise enough. By "single", I meant  that whenever you process a type variable bound, constraints or default, then you enter that block. I didn't want to say "single" in the sense "T: A = B" is a single TypeVariableBlock but rather processing "A" and "B" would create two different TypeVariableBlock.